src/main/java/com/example/airbnbapi/mapper/FetchObject.java

```diff
@@ -0,0 +1,49 @@
+package com.example.airbnbapi.mapper;
+
+import com.example.airbnbapi.Model.Game;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.*;
+
+public class FetchObject {
+
+
+
+
+    public Game[] returnGameList(String jsonFileName) {
+
+
+        if (findJsonFile(jsonFileName)) {
+
+            try {
+                ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+
+                InputStream inputStream = getClass().getResourceAsStream("/" + jsonFileName + ".json");
+
+                return mapper.readValue(inputStream, Game[].class);
+
+
+            } catch (Exception ex) {
+                ex.printStackTrace();
+            }
+        }
+
+        return null;
+    }
+
+    private boolean findJsonFile(String fileName) {
```

Comment:
Consider renaming this to something like `jsonFileExists`? Phrasing a boolean-return method in the style of a question improves readability.